bin/job

```diff
@@ -71,15 +60,17 @@ transformVM() {
   local VMYAML=${DOMXML%.xml}.yaml
 
   # we support only one disk, it needs to be aligned with DNS-1123
-  PVCNAME=$(echo $SRC-disk-1 | sed -r 's/[_.]+/-/g')
-  VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
+  export PVCNAME=$(echo $SRC-disk-1 | sed -r 's/[_.]+/-/g')
+  export VMNAME=$(echo $SRC | sed -r 's/[_.]+/-/g')
 
   # get resources from vm xml
-  CORES=$(xmllint --xpath "/domain/vcpu/text()" $DOMXML)
-  MEMORY=$(xmllint --xpath "/domain/memory/text()" $DOMXML)
-  UNIT=$(xmllint --xpath "string(/domain/memory/@unit)" $DOMXML)
-  BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
+  export CORES=$(xmllint --xpath "/domain/vcpu/text()" $DOMXML)
+  export MEMORY=$(xmllint --xpath "/domain/memory/text()" $DOMXML)
+  export UNIT=$(xmllint --xpath "string(/domain/memory/@unit)" $DOMXML)
+  export BUS=$(xmllint --xpath "string(/domain/devices/disk/target/@bus)" $DOMXML)
+}
 
+createVM() {
```

Comment:
So I think this would be in a new script that we call from the above.

Reply:
OK